src/Illuminate/Container/Container.php

```diff
@@ -285,11 +286,19 @@ public function bindMethod($method, $callback)
      *
      * @param  array|string $method
      * @return string
+     *
+     * @throws InvalidArgumentException
```

Comment:
Please add a leading slash.

Reply:
I have added,
but I`m not sure if it is really needed because we have InvalidArgumentException in use statement